webextensions/api/browserAction.json

```diff
@@ -615,7 +615,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/setBadgeText",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "4"
```

Comment:
✅ https://github.com/chromium/chromium/commit/5132483c2e2775f0e6bcc249e1438cf88dac97a2 landed in [4.0.221.8](https://github.com/chromium/chromium/releases/tag/4.0.221.8) before Chrome 4.0.223.

While we're here:
:x: We can set the sub-feature `setBadgeText.null` to 83, see: https://chromiumdash.appspot.com/commit/09471bf4bedf979b846dc2e51f259602e317ddc7

Reply:
Done in baa6ce8035.